book/common/preamble.tex

```diff
@@ -186,7 +186,15 @@
 	}\xspace%
 }
 
-
+\newenvironment{beforeyouread}{
+Before you read, make sure you are comfortable with the following tasks.
+Please do the ``Quick Check'' problem to see if you are comfortable with each
+ask.
+\begin{itemize}
+}{
+\end{itemize}
+}
+\newcommand\quickcheck[1]{\vspace{-0.5cm}\item[] {\footnotesize \textsc{Quick Check.} \textrm{#1}}}
```

Comment:
Don't use an `\item` for this. Just put in a `\par` at the start. (Using an `\item` will mess things up when we change the item spacing)